test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandler.ValidateTokenAsyncTests_e2e.cs

```diff
@@ -29,7 +30,7 @@ public async Task TestDefaultValidToken()
             };
             CallContext callContext = new CallContext();
 
-            ValidationResult<ValidatedToken> validationResult = await jsonWebTokenHandler.ValidateTokenAsync(token, validationParameters, callContext, default);
+            ValidationResult<ValidatedToken> validationResult = await ((IResultBasedValidation)jsonWebTokenHandler).ValidateTokenAsync(token, validationParameters, callContext, default);
```

Comment:
Instead of having casts everywhere, can we have a private get property returning this?